Formula/nushell.rb

```diff
@@ -33,7 +33,8 @@ class Nushell < Formula
   end
 
   def install
-    system "cargo", "install", "--features", "extra", *std_cargo_args
+    inreplace "./install-all.sh", "cargo install", "cargo install --root=#{prefix}"
```

Comment:
We don't want to omit `std_cargo_args` here.

Reply:
thanks， i will update it later

Reply:
> We don't want to omit `std_cargo_args` here.

But how can I use `std_cargo_args` in `sh` file? I have tried many way, but they don't work, I'm not familiar with ruby, thanks

Reply:
Looks like you were able to resolve this.